http-server/src/main/java/io/micronaut/http/server/HttpServerConfiguration.java

```diff
@@ -598,6 +606,39 @@ public void setNotFoundOnMissingBody(boolean notFoundOnMissingBody) {
         this.notFoundOnMissingBody = notFoundOnMissingBody;
     }
 
+    /**
+     * @return {@code true} if ';' is normal, {@code false} otherwise.
```

Comment:
Please describe it a bit more, it's not clear that this is relating to query parameters.

Reply:
edited, describe it really clear now i guess